tests/phpunit/CiviTest/CiviUnitTestCase.php

```diff
@@ -482,6 +482,8 @@ protected function tearDown() {
 
     $this->cleanTempDirs();
     $this->unsetExtensionSystem();
+    $this->assertEquals(NULL, CRM_Core_DAO::$_nullArray);
```

Comment:
@eileenmcnaughton i think this should be [] not NULL

Reply:
@seamuslee001 looks like jenkins does too :-)

Reply:
Lets see what jenkins thinks now - kinda likely this won't pass